rpcs3/Emu/Io/usio.cpp

```diff
@@ -80,10 +81,12 @@ usb_device_usio::usb_device_usio(const std::array<u8, 7>& location)
 
 	g_fxo->get<usio_memory>().backup_memory.resize(0xB8);
 	g_fxo->get<usio_memory>().last_game_status.resize(0x28);
+	load_backup(g_cfg_vfs.get_device(g_cfg_vfs.dev_usb, "/dev_usb000").path + "usiobackup.bin");
```

Comment:
1. Must work with entwined launches of different games.
2. Must not touch PS3 filesystem.

Reply:
> 1. Must work with entwined launches of different games.
> 
>     2. Must not touch PS3 filesystem.

@elad335
1. Only a few games that adopt USIO save game data in this way. Any other ordinary PS3 games won't be affected by this at all.
2. Any suggestion about where to store the backup file?

Thank you.

Reply:
How does the ps3 do it?

Reply:
> How does the ps3 do it?

I'm not pretty clear about what you are referring to.
In this PR, the backup file will be stored at and read from /dev_usb000/usiobackup.bin

Reply:
I mean how the real ps3 save it?

Reply:
> I mean how the real ps3 save it?

A real USIO board has an on-board battery (just like the CMOS battery, which is used to keep BIOS settings, on your PC's motherboard), and it uses the power from the battery to keep data in its built-in SRAM.
The data of this PR's usiobackup.bin is originally stored in the USIO board's SRAM.

Reply:
> So it's shared with different games on real ps3? If so maybe save it inside /dev_hdd1/caches/somefile. This directory is not actually attached by itself to ps3 filesystem.

Yes. In addition, this kind of games (which adopt a USIO board and save their data in this way) all share the same Title ID "SCEEXE000"; so is it fine to save the backup file at _dev_hdd1/caches/SCEEXE000_SCEEXE000/usiobackup.bin_?

Reply:
Yes, just dev_hdd1/caches/usiobackup.bin.

Reply:
Updated.